client/src/ts/dd_main.ts

```diff
@@ -5,6 +5,21 @@ import vis = require('./visibility');
 import animate = require('./animate');
 import canDefs = require('./can_msg_defs');
 
+// SOC
+
+function mapSocPercent(voltage: number): number {
+  // Use a poor linear approximation for now based on:
+  // https://lygte-info.dk/info/BatteryChargePercent%20UK.html
+  // TODO(ckitagawa): Make this better if time allows.
+  const result = 3.15 * voltage - 373;
```

Comment:
Magic formula? I don't see this anywhere in the page.

Reply:
Added the sheet I used to confluence.